.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,48 @@
+name: Report a Bug
+description: Report a bug here on our issue tracker.
+body:
+- type: "input"
+  id: "labels"
+  attributes:
+    label: "Labels"
+    description: "To automatically label this issue, add the uppercase'd label(s) sourrounded by brackets"
+    value: "[BUG]"
+    placeholder: "[LABEL]"
+
+- type: "input"
+  id: "description"
+  attributes:
+    label: "Description"
+    description: "Give a short and concise description of the bug"
+    placeholder: "Explosive bees do not explode after being fed."
+  validations:
+    required: true
+
+- type: "textarea"
+  id: "steps"
+  attributes:
+    label: "Steps to reproduce"
+    description: "Give a list of steps to recreate this issue"
+    value: "1. "
+    placeholder: |
+       1. Obtain explosive bee from queen bee
+       2. Feed bee any food from the kitchen
+       3. Use explosive bee in hand to prime
+       4. Instead pet the bee
```

Comment:
placeholder does not seem to work when value is prefilled

Reply:
It does not, its meant to direct the user back onto the numbered step list path in the event that they delete the prefiled 1. in the value.